Source/WebDriver/CMakeLists.txt

```diff
@@ -66,13 +66,19 @@ MAKE_JS_FILE_ARRAYS(
 )
 list(APPEND WebDriver_SOURCES ${WebDriver_DERIVED_SOURCES_DIR}/WebDriverAtoms.cpp)
 
-WEBKIT_EXECUTABLE_DECLARE(WebDriver)
+if (ANDROID AND (PORT STREQUAL WPE))
+    set(WebDriver_TARGET_TYPE LIBRARY)
+else ()
+    set(WebDriver_TARGET_TYPE EXECUTABLE)
+endif ()
```

Comment:
Would it be possible to have the default defined here and override it from PlatformWPE.cmake when on android?

Reply:
Not in this case, because the target has to be declared _before_ the platform-specific CMake fragment (`PlatformWPE.cmake` in this case) are included.

Reply:
We have much uglier code than this, so I wouldn't worry about it. Just keep in mind that so long as there is no EWS, we will inevitably find ways to break it by accident.